OpenTaiko/src/Common/ImGuiDebugWindow.cs

```diff
@@ -409,6 +516,22 @@ private static void CTexturePopup(CTexture texture, string label) {
 				}
 			}
 		}
+		private static int CTextureListPopup(IEnumerable<CTexture> texture_list, string label, string id) {
+			if (texture_list == null) return 0;
+			int memory_count = GetTotalMemoryUsageFromCTextureList(texture_list);
+
+			if (ImGui.TreeNodeEx($"{label} Textures: ({texture_list.Count()} / {String.Format("{0:0.###}", GetMemAllocationInMegabytes(memory_count))}MB)###{id}")) {
+				int index = 0;
+				foreach (CTexture tex in texture_list) {
+					CTexturePopup(tex, $"Texture #{index} (Pointer: {(tex != null ? tex.Pointer : "null")})###{id}_{index++}");
+				}
+				ImGui.TreePop();
+			}
+			return memory_count;
+		}
+		private static int CTextureListPopup(ScriptBG script, string label, string id) {
+			return script != null ? CTextureListPopup(script.Textures.Values, label, id) : 0;
```

Comment:
According to the type-annotation script can't be `null`, so `null`-check isn't needed. This is a thing I've noticed in a lot of places in our code and we should probably open up an issue for this.

So in theory this should be 
```csharp
private static int CTextureListPopup(ScriptBG? script, string label, string id) {
                                          // ^ correct type hint
```

Reply:
I've opened up https://github.com/0auBSQ/OpenTaiko/issues/718, so you can resolve this comment.